src/components/Schedule/Schedule.css

```diff
@@ -63,7 +64,7 @@
 	font-family: Rubik;
 	margin: auto;
 	font-weight: bold;
-	font-size: 15px;
+	font-size: 90%;
```

Comment:
Not 1rem? It's 16px so should look very similar to 15px

Reply:
yeah I know, I am trying to make the text changes its size when the size of the button changes but it is not working.

Reply:
Why would the size of the button change?

Reply:
when the screen width changes, it was set like that before so I kept it as it is. But if you want, we can make it fix and doesn't change.

Reply:
I don't think the text needs to change sizes but whatever looks best